packages/kbn-securitysolution-list-utils/src/helpers/index.ts

```diff
@@ -1021,3 +1025,17 @@ export const getMappingConflictsInfo = (field: DataViewField): FieldConflictsInf
   }
   return conflicts;
 };
+
+/**
+ * Given an exceptions list, determine if any entries have an "IS" operator with a wildcard value
+ */
+export const getHasWrongOperator = (items: ExceptionsBuilderReturnExceptionItem[]): boolean => {
```

Comment:
I would rename this function to reflect it is related to wildcars, like `hasWrongOperatorWithWildcard`. 
Otherwise it difficult to say, what this function is doing.